src/node_file.cc

```diff
@@ -81,9 +81,9 @@ using v8::Value;
 #endif
 
 #ifdef __POSIX__
-const char* kPathSeparator = "/";
+constexpr char kPathSeparator = '/';
 #else
-const char* kPathSeparator = "\\/";
+constexpr char kPathSeparator = '\\/';
```

Comment:
This gets casted to `'/'`..I guess `const char*` is fine?

Reply:
@joyeecheung Oh, right, I missed that. Thanks for catching, fixed!